CIP-0072/README.md

```diff
@@ -0,0 +1,413 @@
+- *`action`*: The action that the certificate is asserting. It can take the following values: 
+  - *`REGISTER`*: The certificate is asserting that the dApp is being registered for the first time. 
+  - *`UPDATE`*: The certificate is asserting that the dApp is being updated.
+
+*`rootHash`*: The hash of the metadata entire tree object. This hash is used by clients to verify the integrity of the metadata tree object. When reading a metadata tree object, the client should calculate the hash of the object and compare it with the `rootHash` property. If the two hashes don't match, the client should discard the object. The metadata tree object is a JSON object that contains the dApp's metadata. The metadata tree object is described in the next section.
+
+This hash is calculated by taking the entire metadata tree object, ordering the keys in the object alphanumerically, and then hashing the resulting JSON string using the blake2b-256 hashing algorithm. The hash is encoded as a hex string.
+
+*`signature`*: The signature of the certificate. The signature is done over the blake2b hash of the certificate. The client should use the public key to verify the signature of the certificate. 
+
+## Certificate JSON Schema
+```json
+{
+	"$schema": "https://json-schema.org/draft/2019-09/schema",
+	"$id": "https://example.com/person.schema.json",
+    "title": "Person",
+    "type": "object",
+    "properties": {
+      "subject": {
+        "type": "string",
+        "description": "Can be anything. Description of the registration",
+      },
+      "type": {
+        "type": "object",
+        "description": "Describes the releases, if they are new or an updates. Also states the versioning of such releases",
+        "properties": {
+          "action": {
+          	"type": "string",
+            "description": "Describes the action this certificate is claiming. I.e 'REGISTER', for a new dapp; or 'UPDATE' for a new release"
+          },
+          "releaseNumber": {
+          	"type": "string",
+            "description": "offical version of the release"
+          },
+          "releaseName": {
+          	"type": "string",
+            "description": "Dapp release name"
+          }
+		}
+      },  
+      "rootHash": {
+        "type": "string",
+        "description": "blake2b hash of the metadata describing the dApp"
+      },
+      "signature": {
+        "description": "Age in years which must be equal to or greater than zero.",
+        "type": "object",
+        "properties": {
+			"r": {
+				"type": "string",
+				"description": "hex representation of the R component of the signature"
+			},
+            "s": {
+				"type": "string",
+				"description": "hex representation of the S component of the signature"
+			},
+              
+		},
+		"required": ["r", "s", "algo", "pub"]
+      }
+    },
+    "required": ["subject", "rootHash","type", "signature"]
```

Comment:
Add `schema_version` to the required? also I suggest changing `schema_version` to match how versioning was done in [CIP-25](https://github.com/cardano-foundation/CIPs/blob/master/CIP-0025/README.md?plain=1#L67) - that it is just an integer.

Reply:
Interesting. Could you suggest a change to the CIP? 

CC: @matiwinnetou

Reply:
This is outdated idea from me, not worth pursuing.